example/ingest.py

```diff
@@ -235,22 +316,24 @@ def generate_batches(it, chunk_size):
         yield cur_batch
 
 
+def _calculate_log_level(the_level) -> int:
+    if isinstance(the_level, str):
+        _level_str = str(the_level).lower()
+        if 'debug' in _level_str:
+            return DEBUG
+    return DEFAULT_LOG_LEVEL
+
+
 if __name__ == '__main__':
-    with ProcessPoolExecutor(max_workers=8) as pool:
-        print("Indexing BNL/L'Union articles")
-        futs = []
-        bnl_iter = bnl_load_documents(Path(LUNION_PATH))
-        for idx, batch in enumerate(generate_batches(bnl_iter, 1000)):
-            futs.append(pool.submit(index_documents, batch))
-            print("\r{:05}/{}".format((idx+1)*1000, LUNION_NUM_ARTICLES), end='')
-        for fut in as_completed(futs):
-            fut.result()
-        print("\nIndexing Google 1000 Books volumes")
-        futs = []
-        gbooks_iter = gbooks_load_documents(Path(GOOGLE1000_PATH))
-        for idx, batch in enumerate(generate_batches(gbooks_iter, 4)):
-            futs.append(pool.submit(index_documents, batch))
-            print("\r{:04}/{}".format((idx+1)*4, GOOGLE1000_NUM_VOLUMES), end='')
-        for fut in as_completed(futs):
-            fut.result()
-    print("\n")
+    PARSER = ArgumentParser(description='ingest example data into SOLR')
+    PARSER.add_argument('--log-level', help='like "debug", "info", "error" (default:info)', required=False,
```

Comment:
If we only use logging for debug statements, I think we only need a `--debug` flag to toggle debug logging on.